src/components/table/table-sizes.style.js

```diff
@@ -22,5 +22,11 @@ export default {
     font: '16px',
     padding: '13px',
     inputHeight: '32px'
+  },
+  wrapper: {
+    compact: '24',
```

Comment:
We probably want to keep consistency in the way we declare units; here we have a mixture. Probably better to specify the unit here, rather than in the style string.

Reply:
I've changed it to return a function in the config now as the min-height needs to be calculated dynamically